mycroft/skills/skill_manager.py

```diff
@@ -224,9 +224,14 @@ def setup_finish_interrupt(message):
                 # not implemented
                 services[ser] = True
                 continue
-            elif ser in ["network_skills", "internet_skills"]:
-                # Implemented in skill manager
-                services[ser] = True
+            elif ser in ["skills"]:
+                services[ser] = self.status.check_ready()
+                continue
+            elif ser in ["network_skills"]:
+                services[ser] = self._network_loaded.is_set()
+                continue
+            elif ser in ["internet_skills"]:
+                services[ser] = self._internet_loaded.is_set()
                 continue
             response = self.bus.wait_for_response(
```

Comment:
I was thinking about this when adding the network/internet skills, should we filter this to only check known bus events? That way, adding new settings to config won't break startup and wait for an event that will never reply

Reply:
this is about the internal state of the skill manager, does not really depend on any bus events, what we care about for ready settings is if skill objects are loaded or not

by default ovos-core assumes offline usage and no setup step, so it never blocks waiting for the new events, hence default config including only "skills", if someone changes the config then ensuring the plugin that emits proper events (or a replacement) is out of scope

we should however document the bus events in mycroft.conf comments

Reply:
Right, I think this change makes sense to better check for those conditions being met.

I suppose even a valid setting could cause this to wait forever in some cases (i.e. waiting for `gui` on a headless device), so maybe the `mycroft.conf` comments are sufficient